test/e2e/virtual/apiexport/authorizer_test.go

```diff
@@ -74,10 +74,10 @@ func TestAPIExportAuthorizers(t *testing.T) {
 	orgPath, _ := framework.NewOrganizationFixture(t, server)
 
 	// see https://docs.google.com/drawings/d/1_sOiFZReAfypuUDyHS9rwpxbZgJNJuxdvbgXXgu2KAQ/edit for topology
-	serviceProvider1Path, _ := framework.NewWorkspaceFixture(t, server, orgPath, framework.WithName("service-provider-1"))
-	serviceProvider2Path, _ := framework.NewWorkspaceFixture(t, server, orgPath, framework.WithName("service-provider-2"))
-	tenantPath, tenantWorkspace := framework.NewWorkspaceFixture(t, server, orgPath, framework.WithName("tenant"))
-	tenantShadowCRDPath, tenantShadowCRDWorkspace := framework.NewWorkspaceFixture(t, server, orgPath, framework.WithName("tenant-shadowed-crd"))
+	serviceProvider1Path, _ := framework.NewWorkspaceFixture(t, server, orgPath, framework.WithName("service-provider-1"), framework.TODO_WithoutMultiShardSupport())
```

Comment:
This is tracked in #2713 - would you consider undoing this?

Reply:
Hopefully fixed in #2720. Please undo this change. Going to put a hold on this PR for now.
/hold

Reply:
Yes, this fix has no place in this PR, just trying to get the tests greens as it was too flaky. I will drop it!